packages/create-gatsby/src/__tests__/index.ts

```diff
@@ -0,0 +1,85 @@
+import Enquirer from "enquirer"
+import { installPlugins } from "../install-plugins"
+import { initStarter } from "../init-starter"
+import { createCli, run, IAnswers } from "../"
+
+jest.mock(`../init-starter`)
+jest.mock(`../install-plugins`)
+
+describe(`create-gatsby-app`, () => {
+  let cli: Enquirer<IAnswers>
+
+  beforeEach(() => {
+    cli = createCli()
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it(`creates a project only using the project name`, async () => {
+    cli.on(`prompt`, p => {
+      p.on(`run`, async () => {
+        switch (p.name) {
+          case `project`:
+            p.value = `my-super-project-name`
+            break
+
+          case `features`:
+            await p.keypress(null, { name: `up` })
```

Comment:
Instead of using up can we tab and enter? That's what we'd expect users to do in most cases.